### PR TITLE
Feature : React Client

### PR DESCRIPTION
**Pull request by amolgajewar**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>AlsoViewedItems</title>
+  </head>
+  <body>
+    <div id="app"></div>
```
Comment (tf3) [inline public/index.html:7]: I suggest using a different id (e.g., "also-viewed") to make building the proxy easier.

File: server/app.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
 const express = require('express');
+const path = require('path');
 const db = require('./db/index.js');
 
 const app = express();
 
 app.set('port', 3004);
```
Comment (tf3) [inline server/app.js:7]: I believe it's good practice to set the port number via an environment variable.

File: public/index.html (hunk 1 above)
Comment (amolgajewar) [reply public/index.html:7]: Sure

File: server/app.js (hunk 2 above)
Comment (amolgajewar) [reply server/app.js:7]: Yes I need to set up file to import setup variables

**Pull request merged**